### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by VelzckC0D3**
# Math Magicians -  `React` `ES6` `API`
## ✨ _Requested Implementations_ ✨

✔️ _Set up React Testing Library following the instructions_
✔️ _Write unit test for `operate.js` & `calculate.js`_
✔️ _Created unit test for all the components_
✔️ _Used Jest snapshots to test the components_
✔️ _Used React Testing Library to simulate user interaction_
✔️ _All test passes_

#  💡 _Thank you for your review!_ 💡

### CODE REVIEW

File: src/components/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+import Calculator from './Calculator';
+
+describe('Calculator', () => {
+  test('renders correctly', () => {
+    const { container } = render(<Calculator />);
+    expect(container.querySelector('.inputCont')).toBeTruthy();
+    expect(container.firstChild).toMatchSnapshot();
+  });
+});
```
Comment (OyePriscilla) [inline src/components/Calculator.test.js:10]: - [x] Awesome implementing the snapshots for `Calcuator.js` and `Home.js`, can we please do the same for `Navbar.js` and `Qoutes.js`? You are to use jest snapshots to test all the components. 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @VelzckC0D3 and @Danfall369 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Write unit tests with React Testing Library ✔️ 
Write unit tests with snapshots with Jest ✔️ 
Use of React Testing Library to simulate user interaction ✔️ 
All test passed ✔️ 

## Required Changes ♻️

- [ ] Kindly fix the broken links under the tech stack header of the README file, this will make your repo to be more engaging and professional. 👍 

## Optional suggestions

- [optional] Please I will advise you to change the text under the `tech stack` and `key feature` headers to something informative. 👍  

Untitled


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.test.js (hunk 1 above)
Comment (VelzckC0D3) [reply src/components/Calculator.test.js:10]: Done! @OyePriscilla  thank you, I've already added a more professional and descriptive Readme, as well as the Snapshots to the logic testing components ! 🫡

**Review by levy002 — APPROVED** [review]
## Hello @VelzckC0D3 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**